src/bridge/mod.rs

```diff
@@ -48,10 +48,14 @@ async fn start_neovim_runtime() {
     let nvim = Arc::new(session.neovim);
 
     // Check the neovim version to ensure its high enough
-    match nvim.command_output("echo has('nvim-0.4')").await.as_deref() {
+    match nvim
+        .command_output("echo has('nvim-0.9.2')")
```

Comment:
At this point does it make sense to put the version tab inside the single quotes here into a constant at the top of the file that gets interpolated? That way theres an easy place to update that is known?